MangAdventure/filters.py

```diff
@@ -23,7 +23,7 @@ def title_filter(title: str, klass: Type[FieldListFilter] =
 
     :return: A class that inherits from ``klass``.
     """
-    class _GenericFilter(klass):
```

Comment:
Yet they are not because they are inner classes returned by the function, the function and its return are called but the classes are instanciated from the return by whatever django function actually consumes them. Therefore the code enclosed here is unreachable by coverage